charts/cannon/conf/static/zauth.acl

```diff
@@ -1,17 +1 @@
-a (blacklist (path "/provider")
-             (path "/provider/**")
-             (path "/bot")
-             (path "/bot/**")
-             (path "/i/**"))
-
-b (whitelist (path "/bot")
-             (path "/bot/**"))
-
-p (whitelist (path "/provider")
-             (path "/provider/**"))
-
-# LegalHold Access Tokens
-la (whitelist (path "/notifications")
-              (path "/assets/v3/**")
-              (path "/users")
-              (path "/users/**"))
+a (whitelist (regex "/await"))
```

Comment:
Since cannon only have the `/await` route there is no need to include other rules